community-operators/percona/percona-xtradb-cluster-operator.0.3.0-rc1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,237 @@
+                                "ReadWriteOnce"
+                            ],
+                            "size": "6Gi"
+                        }
+                    },
+                    "proxysql": {
+                        "enabled": true,
+                        "size": 1,
+                        "image": "perconalab/proxysql-openshift:0.2.0",
+                        "resources": {
+                            "requests": {
+                                "memory": "1G",
+                                "cpu": "600m"
+                            }
+                        },
+                        "affinity": {
+                            "antiAffinityTopologyKey": "kubernetes.io/hostname"
+                        },
+                        "volumeSpec": {
+                            "accessModes": [
+                                "ReadWriteOnce"
+                            ],
+                            "size": "2Gi"
+                        }
+                    },
+                    "pmm": {
+                        "enabled": false,
+                        "image": "perconalab/pmm-client:1.17.0",
+                        "serverHost": "monitoring-service",
+                        "serverUser": "pmm"
+                    },
+                    "backup": {
+                        "image": "perconalab/backupjob-openshift:0.2.0",
+                        "schedule": [
+                            {
+                                "name": "sat-night-backup",
+                                "schedule": "0 0 * * 6",
+                                "keep": 3,
+                                "volume": {
+                                    "accessModes": [
+                                        "ReadWriteOnce"
+                                    ],
+                                    "size": "6Gi"
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        ]
+    categories: "Database"
+    certified: "False"
+    description: Percona Xtradb Cluster Operator is a database clustering system for horizontal scaling of MySQL.
+    containerImage: docker.io/perconalab/percona-xtradb-cluster-operator:openshift-4.0
+    createdAt: 2019-02-22T04:21:00Z
+    support: Percona
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: PerconaXtraDBCluster
+      name: perconaxtradbclusters.pxc.percona.com
+      version: v1alpha1
+      displayName: PerconaXtraDBCluster
+      description: Instance of a Percona XtraDB Cluster
+    - kind: PerconaXtraDBBackup
+      name: perconaxtradbbackups.pxc.percona.com
+      version: v1alpha1
+      displayName: PerconaXtraDBBackup
+      description: Instance of a Percona XtraDB Backup
+  links:
+  - name: Percona
+    url: https://www.percona.com/
+  - name: Documentation
+    url: https://percona-lab.github.io/percona-xtradb-cluster-operator/
```

Comment:
This link leads to a 404 @joelanford

Reply:
I think the correct one is -> https://percona.github.io/percona-xtradb-cluster-operator/

Reply:
Will do a PR tomorrow to fix the link, thanks for catching this! They maybe moved it? 🤷‍♀️